src/solution_part2.kt

```diff
@@ -1,3 +1,21 @@
 fun main(){
- print("I love Kotlin!!!!")
+ val Player1HealthPoints = 90
+ val Player2HealthPoints = 40
+
+ if(Player1HealthPoints<=90){
+    if(Player1HealthPoints<50) {
+     println("the player's 1 health is awful")
+    }
+    else {
+     println("the player's 1 health is fine")
+    }
+ }
+ if(Player2HealthPoints<=90) {
+  if (Player2HealthPoints<50) {

```

Comment:
wrong logic 
less than 90 and NOT equal at the same time bigger than 50 
we use range 
when (Player1HealthPoints)
in 50..89->